### ISSUE TITLE
Improve visibility in the asset library

### ISSUE THREAD

**Issue by SoyoTamo** [godotengine/godot-proposals#3715]
### Describe the project you are working on

Nothing in specific, it works fine for any project.

### Describe the problem or limitation you are having in your project

Entering the asset library can be a bit disorienting. Things can only be sorted as new or old and none of that goes a long way in determining the quality or usefulness of the demo or plugin. And looking for tops or recommendations on the internet is better than investigating one by one but it is still not the best way.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

I propose to add two more counts to help demos / plugins that excel in utility and quality above the others to be found by the user. 

Here are the options that I think are more important and that once implemented we will all use to decide what to download and try: 

-Most Downloaded / Used: A download counter could significantly help people who are dedicated to investigating which plugin or assets are at least useful before downloading them, it could help to clean up the jungle that is the asset library, it would also be something that with time it would improve by the feedback. 

-Most Recommended: Add with a count that works with user feedback after using the demo or plugin where it determines (maybe thumbs up / down, idk) if it is recommended or not. Which I feel is more useful and is my personal favorite.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

I don't really know much about programming but I am quite clear that it would not be so difficult to do it.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

It would be part of the editor in conjunction with that of the entire community, so no.

### Is there a reason why this should be core and not an add-on in the asset library?

Yes, because feedback from everyone who uses godot is needed so that this can be useful to everyone. This should be part of the core

**Comment by Calinou** [comment]
Both of those changes require backend changes to the asset library, which won't be made on the current backend as it's in maintenance mode. The [asset library rewrite](https://github.com/Calinou/godot-asset-library-laravel) supports reviews (there's a pull request for [download tracking](https://github.com/Calinou/godot-asset-library-laravel/pull/190) too), but there is no ETA for deploying it to production.

**Comment by SoyoTamo** [comment]
> Both of those changes require backend changes to the asset library, which won't be made on the [current backend](https://github.com/godotengine/godot-asset-library) as it's in maintenance mode. The [asset library rewrite](https://github.com/Calinou/godot-asset-library-laravel) supports reviews (there's a pull request for [download tracking](https://github.com/Calinou/godot-asset-library-laravel/pull/190) too), but there is no ETA for deploying it to production.

You are right. For my part I would consider this change for 4.0 that comes perfect with GDextencion and the incompatibility of the projects. Since all this new material will start to come out from 4.0, things with airtime will not be alienated from this function from the beginning, I am just saying.

**Comment by tobiaswuerth** [comment]
any update on this?

**Comment by Calinou** [comment]
> any update on this?

The asset library will be replaced by the [asset store](https://store-beta.godotengine.org/) in the future, which already features reviews. It does not currently feature download count tracking though.

**Comment by Naros** [comment]
I do want to follow up on Calinou's last post. The asset store does track asset page visits and downloads by version, including an all-time download counter. This information is available in your Dashboard for the asset on the first Overview page. 

When you pair this with the thumbs-up reviews, all proposal items are complete. 🚀
